pygls/lsp/types/language_features/document_highlight.py

```diff
@@ -57,4 +57,4 @@ class DocumentHighlightKind(enum.IntEnum):
 
 class DocumentHighlight(Model):
     range: Range
-    kind: Optional[DocumentHighlightKind] = DocumentHighlightKind.Text
+    kind: Optional[DocumentHighlightKind]
```

Comment:
```typescript
export interface DocumentHighlight {
	/**
	 * The range this highlight applies to.
	 */
	range: Range;

	/**
	 * The highlight kind, default is DocumentHighlightKind.Text.
	 */
	kind?: DocumentHighlightKind;
}
```

It says "The highlight kind, default is DocumentHighlightKind.Text", does it mean that if the server does not send this field, the client will use this value, or the server should send `DocumentHighlightKind.Text` ?

Reply:
If it meant the second of those, then the field would not be optional - right?

So I definitely read this the first way.

Reply:
Probably, ok merging this.